plugins/content_distribution/providers/youtube_api/lib/YoutubeApiDistributionEngine.php

```diff
@@ -30,7 +30,9 @@ class YoutubeApiDistributionEngine extends DistributionEngine implements
 {
 	protected $tempXmlPath;
 	protected $timeout = 90;
-	protected  $processedTimeout = 300;
+	protected $processedTimeout = 300;
+	protected $longProcessedTimeout = 1200;
```

Comment:
The idea was to use this default values if they are not set in configuration

Reply:
I set both properties on the class, then we either override them from config (if config exist) otherwise we keep the default values defined on class.

protected $processedTimeout = 300;
protected $longProcessedTimeout = 1200;

Let me know if I confused it.